modules/guides/pages/upgrades.adoc

```diff
@@ -0,0 +1,102 @@
+= Upgrade
+:description: To benefit from Redpanda Connect's new features and enhancements, upgrade to the latest version.
+
+To benefit from Redpanda Connect's new features and enhancements, including a growing list of connectors, upgrade to the latest version.
+
+== Redpanda Connect versioning
+
+Redpanda Connect uses semantic versioning. Version numbers follow the format `major.minor.patch`:
+
+- `major`: This version may include breaking changes.
+- `minor`: This version only includes backward-compatible new features or enhancements.
+- `patch`: This version includes backward-compatible bug fixes.
+
+For example, version 4.35.1 is the first patch release of bug fixes following the minor release of product enhancements in 4.35.0. Breaking changes would not be released until 5.0.0.
+
+For information about versioning in other Redpanda Self-Managed products, including the `rpk` CLI, see xref:ROOT:upgrade:rolling-upgrade.adoc[Upgrade Redpanda].
+
+== Upgrade Redpanda Connect
+
+The simplest way to upgrade Redpanda Connect is to use the `rpk connect` plugin, which is automatically installed when you start to run `rpk connect` commands. You do not need to upgrade the entire `rpk` CLI. 
+
+For full installation instructions for Redpanda Connect, see xref:guides:getting_started.adoc[Get Started].
+
+=== Prerequisites
+
+- `rpk` CLI installed
+- Review the https://github.com/redpanda-data/connect/releases[release notes^] of the Redpanda Connect version you want to upgrade to.
+
+=== Upgrade to the latest version
+
+Upgrade to the latest version of Redpanda Connect.
+
+. Check the version of Redpanda Connect already installed.
++
+[,bash]
+----
+rpk connect --version
+----
+
+. Upgrade to the latest released version.
+
++
+[,bash]
+----
+rpk connect upgrade
+----
++
+Example output:
++
+```
+Redpanda Connect successfully upgraded from 4.38.0 to the latest version (4.39.0).
+```
+
+For a full list of `rpk connect upgrade` options, run:
+
+[,bash]
+----
+rpk connect upgrade --help
+----
+
+=== Upgrade or downgrade to a specific version
```

Comment:
It might be worth showing that this process (downgrade) can also be achieved with a single command:

```
rpk connect install --connect-version <version> --force
```

Reply:
Updated.